IyiKuyoro/FashionHub-BE
# Convert database to postgress

#### What does this PR do?
This pull request changes the setup of the database from mongo to postgres

#### Description of Task to be completed?
- [x] configure sequelize
- [x] remove mongo and mongoose
- [x] add migrations and models
- [x] modify scripts

#### How should this be manually tested?
- Clone the repo and setup your environment variables
- Setup your databases as per your environment variables
- Run the tests and confirm that they all pass.

#### What are the relevant pivotal tracker stories?
#166264062
